zipkin-ui/test/component_data/default.test.js

```diff
@@ -0,0 +1,32 @@
+import {
+  parseApiQuery
+} from '../../js/component_data/default';
+
+chai.config.truncateThreshold = 0;
+
+describe('parseApiQuery', () => {
+  const should = require('chai').should();
```

Comment:
It shouldn't - expect is already used in other places in the tests. E.g. in traceSummary.test.js:

```javascript
expect(traceSummary([])).to.equal(null);
```

Are you sure it's not some other lint error?